src/Infrastructure/Repository/TaskRepository.php

```diff
@@ -108,6 +109,43 @@ public function findAllIndexedCollection(): array
         return $tasksMap;
     }
 
+    /**
+     * @param \SprykerSdk\Sdk\Infrastructure\Entity\Task $task
+     *
+     * @return \SprykerSdk\SdkContracts\Entity\TaskInterface
+     */
+    protected function changePhpCommand(TaskInterface $task): TaskInterface
+    {
+        $existingCommands = [];
+
+        foreach ($this->existingTasks as $existingTask) {
+            $existingTaskCommands = $existingTask->getCommands();
+
+            if ($existingTask instanceof TaskSetInterface) {
```

Comment:
Do you think a TaskSet can be a subtask for another TaskSet? Is there any chance that this construct makes sense and can arise in any of the possible use cases?